blockstore/apps/api/v1/views/bundles.py

```diff
@@ -0,0 +1,52 @@
+"""
+Views for Bundles and BundleVersions.
+"""
+
+from rest_framework import viewsets
+from rest_framework.generics import get_object_or_404
+
+from blockstore.apps.bundles.models import Bundle, BundleVersion
+
+from ...constants import UUID4_REGEX, VERSION_NUM_REGEX
+from ..serializers.bundles import BundleSerializer, BundleVersionSerializer
+
+
+class BundleViewSet(viewsets.ModelViewSet):
+    """
+    ViewSet for Bundle model.
+    """
+
+    lookup_field = 'uuid'
+    lookup_url_kwarg = 'bundle_uuid'
+    lookup_value_regex = UUID4_REGEX
+
+    queryset = Bundle.objects.all()
+    serializer_class = BundleSerializer
+
+
+class BundleVersionViewSet(viewsets.ReadOnlyModelViewSet):
```

Comment:
Since `BundleVersion.change_description` is mutable, I think we need a `PUT` endpoint too (though the other fields can't be edited.

Reply:
Added todo to LX-53.

Reply:
👍